### DISCUSSION TITLE
How to use nginx to proxy an eastwestgateway of nodeport type

### DISCUSSION THREAD

**Discussion by fizos** [istio/istio#49473] [Q&A]
I have build a Multi-Primary Multi-Network istio clusters, and connect each other with the east-west gateway of NodePort type service.
After assigned the external ip of the eastwest gateway, everything works fine.
When I used  a  nginx proxy to proxy the eastwest gateway nodeport, the traffic breaks down。
this is the nginx conf
```yaml

stream {
    
    server {
        listen 15443;
        proxy_connect_timeout 5s;
        proxy_timeout 20s;
        proxy_pass ${NodeIP}:30322;
    }
}
```
I have already modified the east-west gateway external ip as the nginx ip
Is something wrong? 
How should I config the nginx to proxy the eastwest gateway of nodeport type?

**Comment by lhtkk** [comment]
你咋解决的
